static/app/views/issueList/overview.tsx

```diff
@@ -646,10 +606,18 @@ function IssueListOverview({router}: Props) {
     ]
   );
 
+  useDisableRouteAnalytics(issuesLoading);
+  useRouteAnalyticsEventNames('issues.viewed', 'Issues: Viewed');
   useRouteAnalyticsParams({
+    page: parsePageQueryParam(location, 0),
+    query,
+    num_issues: groups.length,
+    total_issues_count: queryCount,
```

Comment:
are the `num_{category}_issues` not used anymore/not needed?

Reply:
I haven't found any amplitude charts that reference it so I decided not to include. Especially since we are changing the categories soon anyway I figured it probably wasn't needed. Good catch though